WORKSPACE

```diff
@@ -23,7 +23,8 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 git_repository(
     name = "io_bazel_stardoc",
     remote = "https://github.com/bazelbuild/stardoc.git",
-    tag = "0.4.0",
+    commit = "4378e9b6bb2831de7143580594782f538f461180",
+    shallow_since = "1570829166 -0400",
```

Comment:
Tags are mutable, so Bazel complains/tells you to do this if you reference external things by tag.
(`shallow_since` is an optimization; I just copied this from the DEBUG output).